include/mata/nfa/nfa.hh

```diff
@@ -291,38 +258,119 @@ public:
     StateSet post(const StateSet& states, const Symbol& symbol) const;
 
     /**
-     * @brief Expand alphabet by symbols from this automaton to given alphabet
+     * Check whether the language of NFA is empty.
+     * @param[out] cex Counter-example path for a case the language is not empty.
+     * @return True if the language is empty, false otherwise.
+     */
+    bool empty_language(Run* cex = nullptr) const;
```

Comment:
since is_ seems to be used everywhere, I would say is_lang_empty

Reply:
That is the best option in my opinion.